en_ewt-ud-train.conllu

```diff
@@ -160968,10 +160968,10 @@
 3	like	like	SCONJ	IN	_	4	mark	4:mark	_
 4	trying	try	VERB	VBG	VerbForm=Ger	2	advcl	2:advcl:like	_
 5	to	to	PART	TO	_	6	mark	6:mark	_
-6	Make	make	VERB	VB	VerbForm=Inf|Typo=Yes	4	xcomp	4:xcomp	CorrectForm=make
+6	Make	make	VERB	VB	Typo=Yes|VerbForm=Inf	4	xcomp	4:xcomp	CorrectForm=make
```

Comment:
Not sure if you added this but we don't consider it a typo if only the capitalization is weird.

Reply:
I did not know that, for me it is a kind of Typo.

Reply:
If we were only dealing with well-edited genres, then we could mark nonstandard capitalization. But in web data there are people who use all-caps or all-lowercase or other conventions. So we'd rather focus our efforts on typos other than miscapitalization, of which there are plenty!